### ISSUE TITLE
qt5-default is deprecated in Ubuntu 22.04

### ISSUE THREAD

**Issue by dragos-bth** [omnetpp/omnetpp#921]
**Platform**
 - OMNeT++ version: 6.0
 - OS: Ubuntu Server 22.04 (ARM64)

**Describe the bug**
I have tried to install omnetpp-6.0 on Ubuntu Server 22-04. The qt5-default package that pulled in all required qt5 dependencies no longer exists. According to the Ubuntu mailing list it has been deprecated since Ubuntu 21.04 (see [https://www.mail-archive.com/ubuntu-bugs@lists.ubuntu.com/msg5902890.html](https://www.mail-archive.com/ubuntu-bugs@lists.ubuntu.com/msg5902890.html)) . The online installer from QT does not help, because it can only run on x64 architectures.

I have tried ``sudo apt-get install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools``, that does bring in some of the qt5 packages but ``configure`` still complains about  missing moc, uic and rcc.

**Comment by dragos-bth** [comment]
I tried again to install, but this time following the instructions from the InstallGuide for 6.0 (not my own notes from previous installations). In addition, I installed posix_ipc (which ./configure complained about) and adapted the package names for OSG:

```
sudo apt-get install openscenegraph openscenegraph-examples libopenscenegraph-dev
sudo apt install python-is-python3 python3-posix-ipc
```

I was able to compile omnetpp without problems, start it, install INET from the GUI and and compile that without warnings. Also, I was able to run the INET manet showcase without issues

Is nemiver still required for JIT debugging?
